### DISCUSSION TITLE
How to use highcharts3d?

### DISCUSSION THREAD

**Discussion by mohammad-zr** [richardeschloss/nuxt-highcharts#68] [Q&A] [answered]
How to import and use highcharts3d?

with highcharts-vue I used the below code:
```
import Highcharts from 'highcharts';
import highcharts3d from 'highcharts/highcharts-3d';

highcharts3d(Highcharts);
```

but with this package I don't know how to use it

**Comment by richardeschloss** [comment, marked as answer]
https://stackblitz.com/edit/nuxt-starter-5pftra?file=components%2FChart3D.vue
